### PR TITLE
Homepage & Contact Page added

### PR DESCRIPTION
**Pull request by babsmatic1**
Homepage & Contact Page added

### CODE REVIEW

**Review by DeleMike — CHANGES_REQUESTED** [review]
Hi @babsmatic1 , thanks for the PR.
Please you will need to work on this to make some adjustments. 

See screenshots, this is how they appear on my laptop screen


Screenshot (214)
Screenshot (215)

Screenshot (216)
Screenshot (213)

Make changes and just push, I will pull and review again

Please @ade555 , check this out too.

**Comment by babsmatic1** [comment]
Okay thanks for the correction i will also review it in another browser indeed cause most. Of them appear normal and stable.

Thanks for the correction

**Comment by DeleMike** [comment]
> Okay thanks for the correction i will also review it in another browser indeed cause most. Of them appear normal and stable.
> 
> Thanks for the correction

Yeah, you can use Edge browser, that is what I use mostly.

**Review by DesignITAfrica — CHANGES_REQUESTED** [review]

**Review by ade555 — CHANGES_REQUESTED** [review]
I appreciate you using semantic tags in some places, but overall, you still need to improve the accessibility of the site. Here are some resources:

1. [Understanding HTML Techniques for Improving Web Accessibility](https://www.makeuseof.com/essential-html-techniques-for-improving-web-accessibility/).
2. [HTML Accessibility by W3Schools](https://www.w3schools.com/html/html_accessibility.asp)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    </div><br>
+    <section class="Partner">
+        <div class="dishes">
+            <h3 class="explore">Our Trusted<hr width="110px"></h3>
+            <h3 class="explore">Partners</h3>
+        </div><br>
+
+            <div class="filder">
+                
+                <img src="img/partner 1.png" alt="" > &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (ade555) [inline index.html:162]: I have mentioned accessibility several times but you still do not have alt texts for your images?

**Review by ade555 — CHANGES_REQUESTED** [review]

**Comment by babsmatic1** [comment]
I will do that sir

File: index.html
Comment (DeleMike) [inline index.html:1]: @babsmatic1 kindly adjust as Thompson mentioned

**Comment by DeleMike** [comment]
Okay, then send in the preview link. Either comment it or share on the
Slack channel so that we can continue the process

**Comment by DeleMike** [comment]
The intern is not with us.

**Comment by ade555** [comment]
Kindly delete the PR. Thanks.

**Pull request closed without merging**